src/ehrdata/io/csv.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Literal
+
+import pandas as pd
+
+if TYPE_CHECKING:
+    from collections.abc import Iterable
+    from pathlib import Path
+
+    from ehrdata import EHRData
+
+
+def read_csv(
+    filename: Path | str,
+    *,
+    sep: str = ",",
+    index_column: str | None = None,
+    columns_obs_only: Iterable[str] | None = None,
+    format: Literal["flat", "wide", "long"] = "flat",
+    wide_format_time_suffix: str | None = None,
+    long_format_keys: dict[Literal["observation_column", "variable_column", "time_column", "value_column"], str]
+    | None = None,
+    **kwargs,
+) -> EHRData:
+    """Reads a csv file.
```

Comment:
```suggestion
    """Reads a csv file.
```

Read a comma-separated values (csv) file into DataFrame.

is what the pandas docs have: https://pandas.pydata.org/docs/reference/api/pandas.read_csv.html

So let's write it here accordingly to make it obvious that we get an EHRData object. Yes, the return types says it but this function has lots of parameters and one might need to scroll